docs/utilities/parameters.md

```diff
@@ -4,29 +4,40 @@ description: Utility
 ---
 
 
-The parameters utility provides a way to retrieve parameter values from
+The parameters utilities provide a way to retrieve parameter values from
 [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-parameter-store.html), 
 [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/), or [Amazon DynamoDB](https://aws.amazon.com/dynamodb/). 
-It also provides a base class to create your parameter provider implementation.
-
+ 
 **Key features**
 
-* Retrieve one or multiple parameters from the underlying provider
+* Retrieve one or multiple parameters from an underlying provider in a standard way
 * Cache parameter values for a given amount of time (defaults to 5 seconds)
 * Transform parameter values from JSON or base 64 encoded strings
 
 ## Install
-Depending on your version of Java (either Java 1.8 or 11+), the configuration slightly changes.
+In order to provide lightweight dependencies, each parameters module is available as its own
+package:
+
+* **Secrets Manager** - `powertools-parameters-secrets` - [A service that centrally manages the lifecycle of secrets](https://aws.amazon.com/secrets-manager/). Start here if you need to manage secrets only. 
```

Comment:
It's not clear for me these `Start here` ...

Reply:
why not Start here for DDB then ?

Reply:
I would remove the extra blabla, just references the modules and the underlying service they leverage.

Reply:
ACK; it's super redundant with the paragraph above anyway